molecule/commands.py

```diff
@@ -436,6 +436,7 @@ def execute(self):
         print(x)
         print
         self.molecule._print_valid_platforms()
+        print
```

Comment:
Just to be consistent, can we use the print function ``print()`` instead of the keyword. Same goes for line 437.